rectools/models/ease.py

```diff
@@ -65,21 +72,33 @@ class EASEModel(ModelBase[EASEModelConfig]):
     def __init__(
         self,
         regularization: float = 500.0,
-        num_threads: int = 1,
+        num_threads: int = 1,  # TODO: decide. We already have it. But this is actually recommend_cpu_n_threads
```

Comment:
I think it should be `recommend_cpu_n_threads` (`recommend_n_threads`) in the end. 

So, let's do next:
1. Add new param `recommend_n_threads` with correct default value (btw, why do we use 1, not 0?)
2. Change here default value to `None`
3. If this param is not `None` we will use it value, but raise a deprecation warning
4. If this param is `None` we will use `recommend_n_threads` value

Reply:
Fixed. Not sure what to do with tests and configs.
I removed `num_threads` from config. And didn't do any tests for it.